main.go

```diff
@@ -18,21 +18,21 @@ package main
 
 import (
 	"context"
+	"github.com/project-flotta/flotta-operator/internal/common/indexer"
```

Comment:
the formatting of the packages isn't the desired one.
pls check the output of `make all` and compare

Reply:
For some reason `make all` re-formats almost all of the imports and it is different from what's in `main`.

Reply:
Fixed the file